webpack/assets/javascripts/foreman_i18n.js

```diff
@@ -0,0 +1,29 @@
+import Jed from 'jed';
+import { deprecateObjectProperty } from './foreman_tools';
+
+const emptyLocales = {
+  en: { domain: 'app', locale_data: { app: { '': {} } } },
+};
+
+const getLocale = () => {
+  const locale = document.getElementsByTagName('html')[0].lang;
+  return locale.length === 0 ? 'en' : locale.replace(/-/g, '_');
+};
+
+const locales = window.locales || emptyLocales;
+const i18n = new Jed(locales[getLocale()]);
+const cheveronPrefix = window.I18N_MARK ? '\u00BB' : '';
+const cheveronSuffix = window.I18N_MARK ? '\u00AB' : '';
+export const t_ = (...args) =>
+  `${cheveronPrefix} ${i18n.gettext(...args)} ${cheveronSuffix}`;
```

Comment:
If we want to keep the wrapping blank spaces, they should be part of the prefix/suffix so that we don't pollute the translated strings when `I18N_MARK` is turned off.